src/known-organizations/index.js

```diff
@@ -87,6 +88,13 @@ export const KnownOrganizations = {
         image: aragonNetworkImage,
         template: TEMPLATE_MEMBERSHIP,
       },
+      {
+        address: '0x4d2e79c0df917721f082bc1fd7322356962d1eb6',
```

Comment:
Should this actually be [`0x08ac31Dd93c16F1f6c4A0FAE540bA1aD52f581d0`](https://etherscan.io/address/0x08ac31dd93c16f1f6c4a0fae540ba1ad52f581d0)? That's what [etherscan is saying for the ENS record](https://etherscan.io/enslookup?q=sf.aragonid.eth).

Reply:
Fixed in 547f030.